capstone/src/main/webapp/index.html

```diff
@@ -49,7 +49,12 @@ <h3 class="mdc-drawer__title">EpiCURE</h3>
           </section>
           <section class="mdc-top-app-bar__section mdc-top-app-bar__section--align-end">
             <label class="search-label top-app-bar-text">Search</label>
-            <form action="/search/restaurants" method="get" id="search-box">
+            <form
+              action="#"
```

Comment:
Are you overriding the form submission by manually editing the field? Doesn't it still reload the page because it's still a "form submit" action? What are you trying to accomplish here?

Reply:
I'm trying to send the data through the JavaScript pipeline so that it's properly formatted in the front-end. The issue with using the `action` field is that it sends the data to the servlet, but it redirects to the path specified. In that case, it goes to a page with nothing but the raw JSON response body. Is there a better way to handle this such that the URL still has the path "/search/restaurants"?